presto-mongodb/src/test/java/com/facebook/presto/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -270,4 +270,15 @@ public void testRenameTable()
         assertQuery("SELECT value FROM test_rename.tmp_rename_new_table", "SELECT 1");
         assertUpdate("DROP TABLE test_rename.tmp_rename_new_table");
     }
+
+    @Test
+    public void testDeleteDocuments()
+    {
+        assertUpdate("CREATE TABLE test_delete.tmp_delete_rows (value bigint)");
+        MongoCollection<Document> collection = mongoQueryRunner.getMongoClient().getDatabase("test_delete").getCollection("tmp_delete_rows");
+        collection.insertOne(new Document(ImmutableMap.of("value", 1)));
+
```

Comment:
Maybe add another assertQuery before the update to verify if the insert was sucessful or add a presto ctas query to insert

Reply:
Added the assertQuery to check the insert was successful or not.